src/signature.rs

```diff
@@ -303,14 +303,51 @@ impl<'a> VerificationHelper for Helper<'a> {
             0 => anyhow::anyhow!("No valid signature"),
             1 => anyhow::anyhow!("{}", &signature_errors[0]),
             _ => anyhow::anyhow!(
-                "Multiple signature errors: [{}]",
-                signature_errors.join(", ")
+                "Multiple signature errors:\n{}",
+                signature_errors.join("\n")
             ),
         };
         Err(err)
     }
 }
 
+fn pretty_error_chain(err: &anyhow::Error) -> String {
+    let causes = err
+        .chain()
+        .skip(1)
+        .map(|cause| format!("  because: {}", cause));
+    format!(
+        "           {}\n{}",
```

Comment:
```suggestion
        "           {err}\n{}",
```
Also, I wonder if the caller might be a better place to indent the whole text?

Reply:
Thanks for the tip on using names from the scope. Moved the indenting to the caller.